<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gauss</class>
 <widget class="QMainWindow" name="Gauss">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gauss</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="Tabla">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>571</width>
      <height>151</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>Ecuacion 1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Ecuacion 2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Ecuacion 3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Ecuacion 4</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>X1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>X2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>X3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>X4</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>=</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="4">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>235</red>
        <green>235</green>
        <blue>235</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="1" column="4">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>235</red>
        <green>235</green>
        <blue>235</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="2" column="4">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>235</red>
        <green>235</green>
        <blue>235</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="3" column="4">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>235</red>
        <green>235</green>
        <blue>235</blue>
       </color>
      </brush>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="Inicia">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>310</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Calcular</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Limpia">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>310</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Limpiar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Sale">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>310</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>390</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X1:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="show1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>380</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>8</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>390</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X2:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="show2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>380</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>8</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="show4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>430</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>8</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>440</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X3:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="show3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>430</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>8</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>440</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X4:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="valorx1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
    <property name="minimum">
     <double>-999999999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>9999999999.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="Agrega">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ingrese los valores de cada ecuaci√≥n</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="valorx2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
    <property name="minimum">
     <double>-999999999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>9999999999.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="valorx3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
    <property name="minimum">
     <double>-999999999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>9999999999.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="valorx4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>100</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
    <property name="minimum">
     <double>-999999999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>9999999999.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="valori">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>100</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
    <property name="minimum">
     <double>-999999999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>9999999999.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>592</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>Sale</sender>
   <signal>clicked()</signal>
   <receiver>Gauss</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>471</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>541</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
